tools/topology/topology2/include/components/gain.conf

```diff
@@ -154,5 +154,4 @@ Class.Widget."gain" {
 	curve_type		"linear"
 	curve_duration		10
 	init_value		0x7fffffff
-	num_audio_formats 1
```

Comment:
Does this impact SRC. Should this be a separate PR?

Reply:
it is a issue find by src.